_episodes/l1-04-managing_contributions.md

```diff
@@ -0,0 +1,84 @@
+>
+>> ## Solution
+>> 1. On GitHub.com, navigate to your repository and choose your branch which contains your changes from the "Branch" menu.
+>> ![Choose branch]({{ site.baseurl }}/fig/choose_branch.png "Choose branch"){:class="img-responsive"}
+>> 2. From the "Contribute" drop-down menu, choose the "Open pull request" button.
+>> ![Open pull request]({{ site.baseurl }}/fig/pull_request_button.png "Open pull request"){:class="img-responsive"}
+>> 3. From the *base* branch drop-down menu, choose the branch you want your changes to be merged into, and in the *compare* drop-down menu, choose the branch which contains your changes.
+>> ![Choose the base and compare branches from the drop-down]({{ site.baseurl }}/fig/base_compare_drop_down.png "Choose the base and compare branches from the drop-down"){:class="img-responsive"}
+>> 4. After giving a suitable title and description for your pull request, click the "Create pull request" button.
+>> ![Pull request title and description fields and create pull request button]({{ site.baseurl }}/fig/pr_title_description.png "Pull request title and description fields and create pull request button"){:class="img-responsive"}
+> {: .solution}
+{: .challenge}
+
+### 2. Pull request from a forked repository
+Forks are often used in large, open-source projects where you do not have write access to the upstream repository (as opposed to smaller project that you may work on with a smaller team). Proposing changes to someone else's project in this way is called the **fork and pull model**, and follows these three steps;
+1. Fork the repository.
+2. Make the fix.
+3. Submit a pull request to the project owner.
+
+This fork and pull model is a key aspect of open-source projects, allowing community contributions whilst reducing the amount of friction for new contributors in terms of being able to work independently without upfront coordination. Another benefit of forking is that it allows you to use someone else's propject as a starting point for your own idea.
+
+Pull requests from forked repositories work slightly differently to that of pull requests from a branch within a repository.
+
+> ## Forking a repository
+> Let's have a go at forking the book_of_recipes repository on the Imperial College London GitHub organisation.
+> 1. First, navigate to the repository at [https://github.com/ImperialCollegeLondon/book_of_recipes](https://github.com/ImperialCollegeLondon/book_of_recipes) and in the top-right corner click **Fork**.
+> ![Fork button]({{ site.baseurl }}/fig/fork_button.png "Pull request title and description fields and create pull request button"){:class="img-responsive"}
+> 2. Select an owner for the forked repository (if you belong to any GitHub organisations, they will appear here) and give it a suitable name. I am receiving a warning in the screenshot below as I chose myself as the owner and already have a repository with this name; GitHub does not allow you to own two repositories of the same name.
```

Comment:
Maybe for clarity, I would remove the fork from your account and then try this again, so there is no error.